js/5etools-importer.js

```diff
@@ -2,6 +2,10 @@ function d20plusImporter () {
 	d20plus.importer = {};
 
 	d20plus.importer._playerImports = {};
+
+	// Setting this variable to true will grab data from the site, rather than the script's storage
+	d20plus.importer.forceExternalRequests = false;
```

Comment:
so you intend this to be used how? Devs should manually flip this, before building the script, when devving..? While I'm all for better dev tooling, this is not up to par. 
It should either skip setting any UI (because if you know this is here, you know what you're doing), or e.g. update the UI when this is changed in-client (which is definitely too much work/extra implementation for a feature like this).
I would prefer it to be moved out to its own `d20plus.debug. ...` namespace, where we can then put other such flags in future, as required. Document this in e.g. the README, and remove any UI side-effects as unnecessary

Reply:
It can be flipped either in the script or in the console. Where in the readme would you suggest I document this? Going to change importer to debug but not change much, as I don't think this merits a whole new file just yet.